third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -147,6 +151,11 @@ impl TestConfig {
         let mut env = move_compiler_v2::run_checker(options.clone())?;
         let mut ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
 
+        if ok {
+            function_checker::check_functions(&mut env);
+            ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
```

Comment:
Note that this behavior is different from what is in `run_move_compiler`, specifically, we run `check_functions` and `run_inlining` without an intervening check for accumulated errors. I am worried about this divergence hiding behaviors that we are not testing: perhaps add `check_errors` after `check_functions` in `run_move_compiler` to get similar behavior?

Reply:
fjxed